packages/key-management/src/types.ts

```diff
@@ -171,14 +171,16 @@ export interface KeyAgent {
 
 export type AsyncKeyAgent = Pick<KeyAgent, 'deriveAddress' | 'derivePublicKey' | 'signBlob' | 'signTransaction'> & {
   knownAddresses$: Observable<GroupedAddress[]>;
+  getChainId(): Promise<Cardano.ChainId>;
+  extendedAccountPublicKey(): Promise<Cardano.Bip32PublicKey>;
```

Comment:
We should align the naming (there's also `chainId` -> `getChainId` etc.). I think it's better to go with a new name for async method instead of using the name of the synchronous variant.
```suggestion
  getExtendedAccountPublicKey(): Promise<Cardano.Bip32PublicKey>;
```

Reply:
Agree. Updated name here: [99ac5b5](https://github.com/input-output-hk/cardano-js-sdk/pull/560/commits/99ac5b5ca12d3e19dc63ca767c00e19e54371b59)